pmm/pmm2-upgrade-portal-tests.groovy

```diff
@@ -127,6 +152,11 @@ pipeline {
                     which chromium-browser
                     sudo yum -y install mysql
                     sudo ln -s /usr/bin/chromium-browser /usr/bin/chromium
+                    sudo mkdir -p /srv/pmm-qa || :
+                    pushd /srv/pmm-qa
+                        sudo git clone --single-branch --branch \${PMM_QA_GIT_BRANCH} https://github.com/percona/pmm-qa.git .
```

Comment:
Why do we need sudo to clone repo?

Reply:
Without sudo:
`[2022-11-07T10:10:56.007Z] /srv/pmm-qa ~/workspace/pmm2-upgrade-portal-tests-temp
[2022-11-07T10:10:56.007Z] + git clone --single-branch --branch main https://github.com/percona/pmm-qa.git .
[2022-11-07T10:10:56.007Z] Cloning into '.'...
[2022-11-07T10:10:56.007Z] /srv/pmm-qa/.git: Permission denied
script returned exit code 1`